src/test/java/pdfverification/tests/PsvPRSBilingualTests.java

```diff
@@ -33,6 +34,9 @@ public PsvPRSBilingualTests() {
     public void setup() throws Exception {
         byte[] pdfData = pdfGenerationService.generate(htmlGenerator.generate(testCertificate));
         pdfReader = pdfParser.readPdf(pdfData);
+        FileOutputStream fileOutputStream = new FileOutputStream("psvprs.pdf");
```

Comment:
I assume this was put in for testing locally, can you remove from the PR that is going to develop please.